querydsl-sql/src/test/java/com/querydsl/sql/ArrayTypesTest.java

```diff
@@ -13,6 +14,9 @@
 
     @Before
     public void setUp() {
+        System.setProperty("user.timezone", "UTC");
```

Comment:
@timowest 
weird.
I remembered it was failed with testing on querydls-sql module.
but it's gone now.